### ISSUE TITLE
How can I convert the BatchNorm2d.num_batches_tracked's dtype from int64 to float32

### ISSUE THREAD

**Issue by Chinajuedui** [pytorch/pytorch#42964]
## 🚀 Feature

I trained a model with pytorch, the all layers' dtype is float32 , but BatchNorm2d.num_batches_tracked's dtype is int64, this cannot support my model's convertion from pytorch to other mobile infer framework, how can i convert the dtype?

**Comment by gchanan** [comment]
Please ask questions on the forums: https://discuss.pytorch.org/.
